stdlib/tkinter/__init__.pyi

```diff
@@ -2888,7 +2888,7 @@ class Scrollbar(Widget):
     def fraction(self, x: int, y: int) -> float: ...
     def identify(self, x: int, y: int) -> Literal["arrow1", "arrow2", "slider", "trough1", "trough2", ""]: ...
     def get(self) -> tuple[float, float, float, float] | tuple[float, float]: ...
-    def set(self, first: float, last: float) -> None: ...
+    def set(self, float(first), float(last)) -> None: ...
```

Comment:
Hmm, this isn't valid Python syntax :)

Are you sure you fully understand the issue you're trying to fix?

Reply:
yeah just a sec fixing it now

Reply:
No, that wouldn't work. Functions in stub files shouldn't have any bodies.

If you're new to the idea of what a stub file is, please see https://peps.python.org/pep-0484/#stub-files or https://typing.readthedocs.io/en/latest/source/stubs.html#type-stubs for an explanation :)